app/views/works/_meta.html.erb

```diff
@@ -73,6 +73,16 @@
              <%= @work.ip_address %>
           <% end %>
         </dd>
+        <dt class="akismetspam">
```

Comment:
Third time's the charm? This should be `<dt class="akismet">` here on line 76 and `<dd class="akismet">` on line 79. Also, that was supposed to be a little `as` on Line 77, please.